src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2896,6 +2929,32 @@ protected function getDateFormat()
 		return $this->getConnection()->getQueryGrammar()->getDateFormat();
 	}
 
+    /**
+     * Convert a Money to a storable integer.
+     *
+     * @param \browner12\money\Money $value
+     * @return int
+     */
+    public function fromMoney($value)
+    {
+        return $value->subunits();
+    }
+
+    /**
+     * Return value as a Money object.
+     *
+     * @param string $key
+     * @param integer|float|string $value
```

Comment:
if that's the convention okay, but i like to be explicit if I know what the mixed actually can be. thoughts?